source/process/overview.rst

```diff
@@ -25,23 +25,37 @@ Mattermost priorities are managed in Jira tickets, which are created by the core
 Triage
 ~~~~~~
 
-On non-holiday weekdays new tickets are reviewed in a process called "triage", and assigned a Fix Version of "unscheduled", indicating the ticket has enough specificity that it can be assigned to a developer to be completed. If the ticket is a bug, it is assigned the Fix Version for the current release.
+On non-holiday weekdays new tickets are reviewed in a process called "triage", and assigned either a numerical Fix Version for the current release to be prioritized, or a "Triage" Fix Version indicating the ticket has enough specificity that it can be assigned to a developer team to be completed but is not necessarily urgent for the current release.
 
-By default, all tickets are created as public unless they contain sensitive information. The triage process reviews them for sufficient
-specificity. If the ticket is unclear, triage may reassign the ticket back to the original reporter to add more details.
+By default, all tickets are created as public unless they contain sensitive information. The triage process reviews them for sufficient specificity. If the ticket is unclear, triage may reassign the ticket back to the original reporter to add more details.
 
 View `current issues scheduled for the next triage meeting <https://mattermost.atlassian.net/browse/PLT-1203?filter=10105>`__.
 
 Re-triage
 ^^^^^^^^^
 
-If someone feels an existing ticket should be re-examined, they can add "triage" to the Fix Version and it will be routed to the triage team for review at the next meeting.
+If someone feels an existing ticket should be re-examined, they can add "global-triage" to the Fix Version and it will be routed to the triage team for review at the next meeting.
 
 Release Planning
 ~~~~~~~~~~~~~~~~
 
-Release planning sets the "Fix Version" of tickets to one of the upcoming monthly releases. The Fix Version is an estimate of when a
-feature might ship, which may change as the planning process continues, until the ticket is scheduled for a Sprint.
+Release planning sets the "Fix Version" of tickets to one of the upcoming monthly releases. The Fix Version is an estimate of when a feature might ship, which may change as the planning process continues, until the ticket is scheduled for a Sprint. 
+
+Jira Fix Version
+~~~~~~~~~~~~~~~~
+
+Developers, Product Managers, Release and Quality Assurance teams use the Fix Version field for the following pruposes:
+
+ Developers: 
```

Comment:
@grundleborg @jwilander @lieut-data @enahum @levb Please fill in any other use cases where Devs use the Jira Fix Version.

Reply:
For stories, we use them for soft guidelines for what is targeted for what release. For bugs, we use them to know which bugs are priority based on where we are in the release cycle.